openquake/risklib/riskinput.py

```diff
@@ -99,17 +99,19 @@ def assets_by_site(self):
             assets_by_site[ass['site_id']].append(self[i])
         return numpy.array(assets_by_site)
 
-    def values(self):
+    def values(self, aids=None):
         """
+        :param: asset indices where to compute the values (default all)
```

Comment:
Please add the name of the parameter. We could also change the description of the default value.